### DISCUSSION TITLE
References to preprocessing questions for semantic search

### DISCUSSION THREAD

**Discussion by qrdlgit** [qdrant/qdrant#2702] [Q&A]
Looking for references to preprocessing questions for semantic search

For QA semantic search, the question is posed as a .. question.  It seems like that would degrade similarity semantic search as knowledge bases are formatted as questions, are there any common approaches to solving this problem (preprocessing the question to a better format for similarity) that I can reference?

**Comment by azayarni** [comment]
Please ask on our Discord Community Server https://qdrant.to/discord
